src/python/tests/core/build_management/revisions_test.py

```diff
@@ -453,6 +453,37 @@ def test_get_components_list(self):
         revisions_dict, 'libfuzzer_asan_libass')
     self.assertEqual(expected_components_list, actual_components_list)
 
+  def test_get_components_list_no_exact_matches(self):
+    """Test get_components_list (no exact matches)."""
+    data_types.Job(
+        name='libfuzzer_asan_php',
+        environment_string=('PROJECT_NAME = php\n'
+                            'HELP_URL = help_url\n')).put()
+    revisions_dict = {
+        '/src/afl': {
+            'type': 'git',
+            'url': 'https://github.com/google/AFL.git',
+            'rev': '82b5e359463238d790cadbe2dd494d6a4928bff3'
+        },
+        '/src/php-src': {
```

Comment:
Do we want to have a test that has both

/src/php
and /src/php-src ?

Reply:
Done!